gluten-core/src/main/scala/io/glutenproject/expression/ExpressionMappings.scala

```diff
@@ -39,8 +39,8 @@ object ExpressionMappings {
   final val BLOOM_FILTER_AGG = "bloom_filter_agg"
   final val VAR_SAMP = "var_samp"
   final val VAR_POP = "var_pop"
-  final val BIT_AND_AGG = "bitwise_and_agg"
-  final val BIT_OR_AGG = "bitwise_or_agg"
+  final val BIT_AND_AGG = "bit_and"
+  final val BIT_OR_AGG = "bit_or"
```

Comment:
Are they spark names? For Velox backend, maybe we can add the mapping to them in Velox SubstraitParser.

Reply:
Yes they are.
https://github.com/apache/spark/blob/ee9e36dd58c824e4c965a5e2c850f64c59fbc278/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala#L788C19-L790
https://github.com/apache/spark/blob/ee9e36dd58c824e4c965a5e2c850f64c59fbc278/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/bitwiseAggregates.scala#L88

Reply:
Yes, that would be preferred for Velox backend instead of overriding them in Scala side.